tests/Unit/Schema/Directives/BaseDirectiveTest.php

```diff
@@ -22,8 +23,30 @@
  */
 class BaseDirectiveTest extends TestCase
 {
+    public function testGetsModelClassFromDirective(): void
+    {
+        $this->schema .= '
+        type User @modelClass(class: "Team") {
+            id: ID
+        }
+        ';
+
+        $directive = $this->constructFieldDirective('foo: User @dummy');
+
+        $this->assertSame(
+            Team::class,
+            $directive->getModelClass()
```

Comment:
`getModelClass()` is a protected method, but called here from outside. Test works, but IDE claims, that protected method was called. Should we ignore it? Or maybe change to public?

Reply:
I would rather keep it protected.

There is some magic in the test that allows calling protected methods: https://github.com/nuwave/lighthouse/blob/e6cbda2309b5e74b87fa4ef1c1311529bdd0c049/tests/Unit/Schema/Directives/BaseDirectiveTest.php#L173

As the BaseDirective is such a common class to extend, i think its internal API should be kept stable.

Reply:
Ok nice.